charts/langsmith/templates/_helpers.tpl

```diff
@@ -272,6 +272,13 @@ Template containing common environment variables that are used by several servic
 {{- if .Values.config.blobStorage.enabled }}
 - name: FF_S3_STORAGE_ENABLED
   value: {{ .Values.config.blobStorage.enabled | quote }}
+- name: FF_BLOB_STORAGE_ENABLED
```

Comment:
Will the old values still work?

Reply:
old values are used for now, yeah. will start using the new ones after this is released

Reply:
sorry i mean like for people not using helm, will the old variables still be read

Reply:
ah right, probably worth respecting the old values until 0.9?

Reply:
yea though i guess that can just be a backend thing to read both